source/blog/2016-11-02-ovirt-provider-ovn.md

```diff
@@ -51,6 +51,7 @@ After installing the driver and OVN, the OVN-controller must be configured. This
     vdsm-tool ovn-config <OVN central server IP> <local IP used for OVN tunneling>
 
 The second parameter (local IP used for OVN tunneling) can be the IP address of the ovirtmgmt interface on the host, and should be reachable by OVN hosts and central server.
+vdsm-tool will also configure ovn-controller with the PKI required to safely connect to the ovn central server.
```

Comment:
Added with some modifications:
- we use the vdsm key/cert for SSL.
- ovn-controller only connects to the south db
- ovn-controller is the part installed on the hosts, there is not central ovn-controller (misleading, I know)

https://gerrit.ovirt.org/#/c/74895